### ISSUE TITLE
Bug ? Get collection with filters and several IRI

### ISSUE THREAD

**Issue by AlexandreHagen** [api-platform/core#775]
Hi 

If I call: 

```
GET: votes?decision=/decision/2&creator=/users/1
```

![capture d ecran 2016-09-28 a 16 57 28](https://cloud.githubusercontent.com/assets/2535878/18918952/ad32ea62-859c-11e6-82fe-8919ee9a7d03.png)

I got that error: 

```
An exception occurred while executing 'SELECT count(DISTINCT v0_.id) AS sclr_0 FROM vote v0_ INNER JOIN decision d1_ ON v0_.decision_id = d1_.id WHERE d1_.id = ?' with params ["\/decisions\/414&creator=\/users\/43"]:

SQLSTATE[22P02]: Invalid text representation: 7 ERROR: invalid input syntax for integer: "/decisions/414&creator=/users/43"
```

And the route used is: 
![capture d ecran 2016-09-28 a 16 32 52](https://cloud.githubusercontent.com/assets/2535878/18917860/481c48c4-8599-11e6-9882-609d3eca8da6.png)

However, if I do:

```
GET: votes?decision=2&creator=1
```

It's working 
Any idea of why I can't use filters with several IRI ?

**Comment by AlexandreHagen** [comment]
My bad, one "s" is missing to ```/decisions/2```

**Comment by teohhanhui** [comment]
But the error you're getting is similar to bug #292

No exception should be thrown for invalid filter values.

**Comment by Simperfit** [comment]
@AlexandreHagen Could you test that again ? Do you want to tu submit a PR ?

**Comment by soyuka** [comment]
Closing as fixed #1844 is going to be merged.
